### DISCUSSION TITLE
Make NumeRe run on Macs

### DISCUSSION THREAD

**Discussion by semibroiled** [numere-org/NumeRe#163] [Ideas]
@numeredev 

Is there any vested interest on making it run on Macs? I am normally on linux or Mac and not windows, I can imagine it being interesting for lots of potential users in academics or enterprise if it can run there

**Comment by numeredev** [comment]
In principle, the interest is there. What we lack is the ability to test on such a system. If you're interested, we can try to achieve that. But be aware that this will be a longer journey.

**Reply by semibroiled** [reply]
I can test it on my Mac so thats not a problem. But long journey case is true, if you're interested Erik, sometime next year the IEEE Student branch in Darmstadt will be founded, many of whom are interested in contributing to open source. I can pitch this there. I just returned my hardware to Invensity, so cant communicate on teams...its anyways preferable for me to do it via discord on my devices

**Reply by numeredev** [reply]
Yeah, that could be a solution. It's not *that* problematic but will take some time as the most Windows-heavy things are the filesystem-specifics.

**Reply by ajbozdar** [reply]
> Yeah, that could be a solution. It's not _that_ problematic but will take some time as the most Windows-heavy things are the filesystem-specifics.

The tides have turned for the Dramstadt IEEE student branch. :smirk:
